hist/histv7/inc/ROOT/RHistImpl.hxx

```diff
@@ -418,27 +657,81 @@ public:
    /// Normalized axes access, converting from actual axis type to base class
    const RAxisBase &GetAxis(int iAxis) const final { return *std::apply(Internal::GetAxisView<AXISCONFIG...>, fAxes)[iAxis]; }
 
-   /// Gets the bin index for coordinate `x`; returns -1 if there is no such bin,
+   /// 
+   std::vector<int> GetBinType(const CoordArray_t &x) const
```

Comment:
```suggestion
   std::array<int,2> GetBinType(const CoordArray_t &x) const
```

Reply:
Add `enum class` for "underflow, overflow, content" instead of "-1, -2, 0" in returned array index 1?